src/SFA.DAS.ApprenticeFeedback.Data.UnitTests/ApprenticeFeedbackRepository/WhenGettingApprenticeFeedbackTargetById.cs

```diff
@@ -0,0 +1,51 @@
+﻿using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApprenticeFeedback.Data.UnitTests.DatabaseMock;
+using SFA.DAS.ApprenticeFeedback.Domain.Entities;
+using SFA.DAS.ApprenticeFeedback.Domain.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeFeedback.Data.UnitTests.ApprenticeFeedbackRepository
+{
+    public class WhenGettingApprenticeFeedbackTargetById
+    {
+        private Mock<IApprenticeFeedbackDataContext> _dbContext;
+        private Repository.ApprenticeFeedbackRepository _repository;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _dbContext = new Mock<IApprenticeFeedbackDataContext>();
+            _repository = new Repository.ApprenticeFeedbackRepository(_dbContext.Object);
+        }
+
+        [Test, MoqAutoData]
+        public async Task Then_ApprenticeFeedbackTargetIsSuccessfullyReturned(Guid mockId, Guid mockApprenticeId, long mockApprenticeshipId, int mockStatus, long mockUkprn, string mockProviderName, string mockStandardUId, string mockStandardName, ICollection<FeedbackTransaction> mockEmailTransactions, ApprenticeFeedbackTarget mockApprenticeFeedbackTarget)
+        {
+
+            _dbContext.Setup(s => s.ApprenticeFeedbackTargets).ReturnsDbSet(new List<Domain.Entities.ApprenticeFeedbackTarget>()
```

Comment:
Can this not be
``` _dbContext.Setup(s => s.ApprenticeFeedbackTargets).ReturnsDbSet(new List<Domain.Entities.ApprenticeFeedbackTarget>()
            { mockApprenticeFeedbackTarget}```

then the bottom part can just be the equivalent assertion? unsure what the gain is by doing the update line by line?